slatedb/src/db/builder.rs

```diff
@@ -191,6 +203,8 @@ impl<P: Into<Path>> DbBuilder<P> {
     /// Builds and opens the database.
     pub async fn build(self) -> Result<Db, SlateDBError> {
         let path = self.path.into();
+        // TODO: proper URI generation, for now it works just as a flag
+        let wal_object_store_uri = self.wal_object_store.as_ref().map(|_| String::new());
```

Comment:
Note: I'm working on https://github.com/apache/arrow-rs-object-store/pull/348 so we can use `ObjectStoreRegistry` to do this. For a subsequent PR.